_layouts/dataset.html

```diff
@@ -34,11 +37,11 @@
     <h2>Resources</h2>
     <ul>
       {% for resource in page.resources %}
-      <li>
-        <a href="{{ resource.url }}">{{ resource.name }}</a>
-        {% if resource.format %}({{ resource.format}}){% endif %}
-        {% if resource.description %}<div class="resource-description">{{ resource.description }}</div>{% endif %}
-      </li>
+        <li>
+          <a href="{{ resource.url }}">{{ resource.name }}</a>
+          {% if resource.format %}<span class="label label-pill label-default">{{ resource.format }}</span>{% endif %}<br>
+          {% if resource.description %}<div class="resource-description"><code>{{ resource.description }}</code></div>{% endif %}
```

Comment:
try not to add too much in main.css as closer to native bootstrap/jekyll themeing seemed simpler